indico/modules/events/abstracts/models/reviews.py

```diff
@@ -0,0 +1,230 @@
+        nullable=True,
+        index=True
+    )
+    abstract = db.relationship(
+        'Abstract',
+        lazy=False,
+        backref=db.backref(
+            'judgments',
+            lazy='dynamic'
+        )
+    )
+    user = db.relationship(
+        'User',
+        lazy=False,
+        backref=db.backref(
+            'abstract_judgments',
+            lazy='dynamic'
+        )
+    )
+    track = db.relationship(
+        'Track',
+        lazy=False,
+        foreign_keys=[track_id],
+        backref=db.backref(
+            'abstract_reviews',
+            lazy='dynamic'
+        )
+    )
+    proposed_track = db.relationship(
+        'Track',
+        lazy=False,
+        foreign_keys=[proposed_track_id],
+        backref=db.backref(
+            'abstract_reviews',
+            lazy='dynamic'
+        )
+    )
+    proposed_type = db.relationship(
+        'ContributionType',
+        lazy=False,
+        backref=db.backref(
+            'abstract_reviews',
+            lazy='dynamic'
+        )
+    )
+
+    @return_ascii
+    def __repr__(self):
+        return format_repr(self, 'id', 'proposed_action', abstract=self.abstract, user=self.user)
+
+    @property
+    def rating(self):
+        return self.ratings
+
+
+class AbstractReviewRating(db.Model):
+    __tablename__ = 'abstract_review_ratings'
+    __table_args__ = (db.UniqueConstraint('abstract_review_id'),
+                      {'schema': 'event_abstracts'})
+
+    id = db.Column(
+        db.Integer,
+        primary_key=True
+    )
+    question_id = db.Column(
+        db.Integer,
+        db.ForeignKey('event_abstracts.abstract_questions.id'),
+        nullable=False
+    )
+    review_id = db.Column(
+        db.Integer,
+        db.ForeignKey('event_abstracts.abstract_reviews.id'),
+        index=True,
+        nullable=False
+    )
+    value = db.Column(
+        db.Integer,
+        nullable=False
+    )
+    question = db.relationship(
+        'AbstractReviewQuestion',
+        lazy=False,
+        backref=db.backref(
+            'ratings',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+    review = db.relationship(
+        'AbstractReview',
+        lazy=False,
+        backref=db.backref(
+            'ratings',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+
+class AbstractReviewQuestion(db.Model):
+    __tablename__ = 'abstract_review_questions'
+    __table_args__ = (db.UniqueConstraint('abstract_review_id'),
+                      {'schema': 'event_abstracts'})
+
+    id = db.Column(
+        db.Integer,
+        primary_key=True
+    )
+    event_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.events.id'),
+        nullable=False
+    )
+    text = db.Column(
+        db.Text,
+        nullable=False
+    )
+    min_rating = db.Column(
+        db.Integer,
+        nullable=False,
+        default=0
+    )
+    max_rating = db.Column(
```

Comment:
Those are not set per question but per event (-> SettingsProxy) unless we want to change this, but in the mockups it was still global.